src/test/resources/org/opentripplanner/middleware/otp/response/transitLeg.json

```diff
@@ -0,0 +1,79 @@
+{
+  "startTime": 1597106580000,
+  "endTime": 1597107180000,
+  "departureDelay": 0,
+  "arrivalDelay": 0,
```

Comment:
It would be useful to have additional mock responses with delays for testing notifications for delays.

Reply:
More tests to be developed later.